examples/docker-compose/.env.example

```diff
@@ -1,5 +1,31 @@
+# Change all these variables before first creating the containers
 POSTGRES_PASSWORD=secret-pg-password-never-use-this-value
-HASURA_GRAPHQL_ADMIN_SECRET=nhost-admin-secret
-HASURA_GRAPHQL_JWT_SECRET='{"type":"HS256", "key":"5152fa850c02dc222631cca898ed1485821a70912a6e3649c49076912daa3b62182ba013315915d64f40cddfbb8b58eb5bd11ba225336a6af45bbae07ca873f3","issuer":"hasura-auth"}'
+HASURA_GRAPHQL_ADMIN_SECRET=change-me
+# The jwt secret key can be generated with `openssl rand -hex 32`
+HASURA_GRAPHQL_JWT_SECRET='{"type":"HS256", "key":"change-this-to-32-char-value","issuer":"hasura-auth"}'
 STORAGE_ACCESS_KEY=storage-access-key-never-use-this-value
 STORAGE_SECRET_KEY=storage-secret-key-never-use-this-value
+
+# The following HOST and URL env variables are separated since the HOST variables are used to define the allowed hosts to the traefik services in the docker-compose.yaml
+
+# Public (if making your Hasura publicly accessible) or private (change to localhost) hostname for the Hasura API/proxy for the Nhost dashboard to send requests to 
+PROXY_HOST=proxy.my-nhost.com
+# Change to http://${PROXY_HOST}:1337 if using localhost
+PROXY_URL=https://${PROXY_HOST}
+
+# Public (if making your Nhost dashboard publicly accessible) or private (change to localhost) hostname for the Nhost dashboard for the proxy to allow requests from
+NHOST_HOST=example-url.my-nhost.com
+
+# Environment variables for the Nhost Dashboard. See dashboard/.env.example for default values. The defualt values are changed to work with our traefik setup created with the docker-compose
+
+# URL for the migrations API running from running `hasura-console` locally. It runs on port 9693 so change to http://localhost:9693 if you are running a private instance 
```

Comment:
It seems the default port for `hasura console` is 9695, not 9693, is that unique to my system?